FSJ_django20_project/FSJ/models_award.py

```diff
@@ -1,35 +1,21 @@
 import uuid
 from django.utils.translation import gettext_lazy as _
 from django.db import models
+from .models_yearofstudy import *
 import datetime
 
 class Award(models.Model):
 	#All awards will have these attributes in common, will be able to select multiple years of study
-	FIRST = 1
-	SECOND = 2
-	THIRD = 3
-	FOURTH = 4
-	FIFTH = 5
-	YEAR_CHOICES = (
-		(FIRST, _('First')),
-		(SECOND, _('Second')),
-		(THIRD, _('Third')),
-		(FOURTH, _('Fourth')),
-		(FIFTH, _('Fifth'))
-	)
 	awardid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 	award_name = models.TextField(verbose_name = _("Award Name"))
 	description = models.TextField(verbose_name = _("Description"))
 	value = models.TextField(verbose_name = _("Value"))
 	programs = models.TextField(verbose_name = _("Programs"))
-	years_of_study = models.IntegerField(choices = YEAR_CHOICES, verbose_name = _("Years of Study"))
+	years_of_study = models.ManyToManyField(YearOfStudy)
```

Comment:
May wish to define a behaviour for on delete here too? If an award is deleted, what happens to the years (probably nothing) but if a year that's associated with an award if deleted?